docs/source/getting-started/security-basics.rst

```diff
@@ -115,29 +115,29 @@ using the ``CONFIGPROXY_AUTH_TOKEN`` environment variable:
 
 This environment variable needs to be visible to the Hub and Proxy.
 
-Default if token is not set
+Default if Token is Not Set
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you don't set the Proxy authentication token, the Hub will generate a random
-key itself, which means that any time you restart the Hub you **must also
+key itself. This means that anytime you restart the Hub, you **must also
 restart the Proxy**. If the proxy is a subprocess of the Hub, this should happen
 automatically (this is the default configuration).
 
 .. _cookie-secret:
 
-Cookie secret
+Cookie Secret
 -------------
 
-The cookie secret is an encryption key, used to encrypt the browser cookies
+The cookie secret is an encryption key, used to encrypt the browser cookies,
 which are used for authentication. Three common methods are described for
 generating and configuring the cookie secret.
 
-Generating and storing as a cookie secret file
+Generating and Storing as a Cookie Secret File
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The cookie secret should be 32 random bytes, encoded as hex, and is typically
-stored in a ``jupyterhub_cookie_secret`` file. An example command to generate the
-``jupyterhub_cookie_secret`` file is:
+stored in a ``jupyterhub_cookie_secret`` file. Below, is an example of a command to generate the
```

Comment:
No comma here. Let's also drop 'of a', 'example command' is still fine.

Reply:
Thank you.